test/test_broker.hpp

```diff
@@ -1538,17 +1580,19 @@ class test_broker {
             return con.get() == &ep;
         }
 
-        MQTT_NS::buffer topic;
+        sub_con_online_map& sco_map;
         con_sp_t con;
+        MQTT_NS::buffer topic_filter;
         MQTT_NS::subscribe_options subopts;
         MQTT_NS::optional<std::size_t> sid;
+        MQTT_NS::optional<sub_con_online_map::handle> handle; // to efficient remove
```

Comment:
I noticed that no optional required. It is initialized at the constructor initializer list.